harvard-lts/fits
# Update to JHOVE 1.26.0; configuration changes

**Resolves Issue**: ([Upgrade to jhove 1.26.0](https://github.com/harvard-lts/fits/issues/322))

# What does this Pull Request do?
- Upgrade the embedded JHOVE tool to 1.26.0
- Changed the new `<consolidate-first-identity>false</consolidate-first-identity>` value in the fits.xml config file to 'false' so that metadata from additional identity elements are included. This results in test results for PDF files similar to those before the JHOVE upgrade.
- As a result of the above configuration change, it was necessary to add a few file type exclusions to 3 different tools to eliminate unnecessary or erroneous metadata in some test cases.
- Updated expected test output as a result of the above configuration changes. 

# How should this be tested?

- Confirm the building of FITS: `mvn clean package -DskipTest`
- Run tests using Docker as indicated in the comments at the top of [this](https://github.com/harvard-lts/fits/blob/jhove-1.26.0_updated/docker/Dockerfile-test) file.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? yes
- integration tests? yes

# Interested parties
@awoods
